umd-fcrepo-event-router/src/main/cfg/edu.umd.lib.fcrepo.camel.router.cfg

```diff
@@ -19,3 +19,6 @@ batch.jms.priority=4
 
 # Skip indexing of "/pcdm" node on batch mode
 batch.skip.pcdm.container=true
```

Comment:
Do we still need `batch.skip.pcdm.container` configuration separately? The `batch.skip.paths` can be configured to skip `/pcdm` as well, right?